src/java/org/apache/cassandra/tools/nodetool/stats/StatsTableComparator.java

```diff
@@ -338,6 +338,47 @@ else if (sortKey.equals("table_name"))
         {
             return sign * stx.tableName.compareTo(sty.tableName);
         }
+        else if(sortKey.equals("QueryLatency"))
+        {
+            result = compareDoubles(stx.QueryLatencyMs, sty.QueryLatencyMs);
+        }
+        else if(sortKey.equals("PostFilteringReadLatencyms"))
+        {
+            result = compareDoubles(stx.PostFilteringReadLatencyms, sty.PostFilteringReadLatencyms);
+        }
+        else if(sortKey.equals("disk_used_bytes"))
+        {
+            return sign * ((String) stx.DiskUsedBytes).compareTo((String) sty.DiskUsedBytes);
+        }
+        else if(sortKey.equals("SSTableIndexesHit"))
+        {
+            return compareDoubles(stx.SSTableIndexesHit, sty.SSTableIndexesHit);
+        }
+        else if(sortKey.equals("IndexSegmentsHit"))
+        {
+            return compareDoubles(stx.IndexSegmentsHit, sty.IndexSegmentsHit);
+        }
+        else if(sortKey.equals("RowsFiltered"))
+        {
+            return compareDoubles(stx.RowsFiltered, sty.RowsFiltered);
+        }
+        else if(sortKey.equals("TotalQueryTimeouts"))
+        {
+            return sign * ((Long) stx.TotalQueryTimeouts).compareTo((Long) sty.TotalQueryTimeouts);
+        }
+        else if(sortKey.equals("TotalIndexCount"))
+        {
+            return sign * ((int) stx.TotalIndexCount);
+        }
+        else if(sortKey.equals("TotalQueryableIndexCount"))
+        {
+            return sign * ((int) stx.TotalQueryableIndexCount);
```

Comment:
For `TotalIndexCount` and `TotalQueryableIndexCount`, are we actually comparing anything here?